README.md

```diff
@@ -266,7 +400,7 @@ c = Client(host='grpc://0.0.0.0:51000')
 c.post('/', Document())
 ```
 
-This simple refactoring allows developers to write an application in the client-server style. The separation of Flow YAML and Executor Python file does not only make the project more maintainable but also brings scalability and concurrency to the next level:
+The separation of Flow YAML and Executor Python file does not only make the project more maintainable but also brings scalability and concurrency to the next level:
 - The data flow on the server is non-blocking and async. New request is handled immediately when an Executor is free, regardless if previous request is still being processed.
 - Scalability can be easily achieved by the keywords `replicas` and `needs` in YAML/Python. Load-balancing is automatically added when necessary to ensure the maximum throughput.
```

Comment:
needs has nothing to do with scalability

Reply:
I updated the scaling section so we highlight replicas and dynamic batching.
Dynamic batching was successfully validated to improve throughput

Reply:
Dynamic batching enabled and validated